ui/src/app/shared/services/workflows-service.ts

```diff
@@ -261,14 +261,10 @@ export const WorkflowsService = {
         }
 
         // return archived log if main container is finished and has artifact
-        return from(this.isWorkflowNodePendingOrRunning(workflow, nodeId)).pipe(
```

Comment:
oh nevermind, this is actually an async function that can also error, so some of this logic is actually different.

I'm not sure that `isWorkflowNodePendingOrRunning` actually needs a `get` request in the first place though, so maybe will revisit both together to simplify